installer/templates/phx_umbrella/apps/app_name_web/config/config.exs

```diff
@@ -20,7 +20,7 @@ config :<%= @web_app_name %>, <%= @endpoint_module %>,
 # Configure esbuild (the version is required)
 config :esbuild,
   version: "0.17.11",
-  default: [
+  "<%= @web_app_name %>": [
```

Comment:
I think we don't need the quotes, the web app name should be a valid identifier.